api/app/controllers/webhook.controller.js

```diff
@@ -0,0 +1,48 @@
+const db = require("../sequelize/models");
+const User = db.user;
+const Review = db.review;
+const Transcript = db.transcript;
+const Op = db.Sequelize.Op;
+
+
+exports.create = async (req, res) => {
+    // const action = req.headers["x-github-event"];
+    const pull_request = req.body;
```

Comment:
can we also account for cases where the `req.body` is undefined or does not contain `pull_request`?